src/boot/lib/mexpr.ml

```diff
@@ -119,6 +119,16 @@ let builtin =
   ; ("mapMem", f (CmapMem None))
   ; ("mapMap", f (CmapMap None))
   ; ("mapMapWithKey", f (CmapMapWithKey None))
+  ; ("tensorCreate", f (CtensorCreate None)) (* MCore intrinsics: Tensors *)
+  ; ("tensorGetExn", f (CtensorGetExn None))
+  ; ("tensorSetExn", f (CtensorSetExn (None, None)))
+  ; ("tensorRank", f CtensorRank)
+  ; ("tensorShape", f CtensorShape)
+  ; ("tensorReshapeExn", f (CtensorReshapeExn None))
+  ; ("tensorCopyExn", f (CtensorCopyExn None))
+  ; ("tensorSliceExn", f (CtensorSliceExn None))
+  ; ("tensorSubExn", f (CtensorSubExn (None, None)))
+  ; ("tensorIteri", f (CtensorIteri None))
   ; ("mapBindings", f CmapBindings) (* MCore intrinsics: Boot parser *)
```

Comment:
I believe `"mapBindings"` should be placed with the other map intrinsics, i.e. before the tensor intrinsics.

Reply:
Good spotting!